frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -233,6 +234,10 @@ const getDeploymentConfigVersion = (obj: K8sResourceKind): number => {
   return version && parseInt(version, 10);
 };
 
+const getDeploymentConfigName = (obj: K8sResourceKind): string => {
+  return getAnnotation(obj, DEPLOYMENT_CONFIG_NAME_ANNOTATION);
```

Comment:
You should use `metadata.ownerReferences` instead of the annotation.

Reply:
fixed in https://github.com/openshift/console/pull/3627/commits/e0d1c431754ec9c3ea4b8b29025478d761bc501f